shopinvader/wizards/shopinvader_category_binding_wizard.py

```diff
@@ -34,22 +34,35 @@ def default_get(self, fields_list):
             })
         return result
 
+    def _bind_categories(self, backend, lang, categories, parent_ids):
```

Comment:
@Cedric-Pigeon ` parent_ids=None`):` Moreover it would be nice to have a description saying that categories are recursively binded.

Reply:
Done